server/middleware/isEmployee.js

```diff
@@ -19,7 +19,8 @@ const isEmployee = (req, res, next) => {
 
   try {
     const { id } = jwt.verify(token, process.env.JWTSECRET);
-    if (!User.isUserExist(id)) {
+    const user = await userModel.select('*', 'id=$1', [id]);
+    if (!user.length) {
       return res.status(UNAUTHORIZED).send({
         status: UNAUTHORIZED,
         error: 'Awww, Snap!..Such kind of access token does not match any employee!',
```

Comment:
@hakoemmy ,  please create an helper file which handles all the responses either success or error.
the helper files should be such that you pass in the required parameters and it does the needful.
i.e
```js
return responsehandler.success(statusCode, 'message', payload)
```
i.e
```js
return responsehandler.error(404, 'item not found')
```

Reply:
Well detailed. Let me fix this ASAP